packages/react-test-renderer/src/ReactTestRenderer.js

```diff
@@ -14,10 +14,13 @@ import type {Instance, TextInstance} from './ReactTestHostConfig';
 import * as TestRenderer from 'react-reconciler/inline.test';
 import {batchedUpdates} from 'events/ReactGenericBatching';
 import {findCurrentFiberUsingSlowPath} from 'react-reconciler/reflection';
+import {getLazyComponentTypeIfResolved} from 'react-reconciler/src/ReactFiberLazyComponent';
```

Comment:
We should never deep link in renderers. This must go onto reflection if you want it. However, better yet is probably to duplicate the logic here. "But then it'll be inconsistent if we change something!" Yea, but that's only because this whole renderer is already a fork of a bunch of internal logic that will be inconsistent all the time.

Reply:
Sorry, that was vscode's auto importer. Forget to fix. It works well if you already imported from that module, not as well for new imports.